tables-example/README.md

```diff
@@ -0,0 +1,17 @@
+# HubSpot Getting Started Project Template
```

Comment:
Same comment as the other PR re: getting Product & UX content for this README so it matches other examples